### PR TITLE
Update instructions for updating single clusters

### PR DESCRIPTION
**Pull request by kylos101**
## Description

KOTS does not stop Gitpod workspaces prior to deleting the Gitpod installation. As a result, it results in data loss when Gitpod is installed if workspaces are actively running during the upgrade window. Let's warn customers about this, instructing them to stop workspaces before proceeding with an upgrade.

## Related Issue(s)

Related to # https://github.com/gitpod-io/gitpod/issues/13147

## How to test

1. Install a self hosted environment July or earlier would be great.
2. Start some workspaces.
3. Use the August KOTS to upgrade

## Release Notes

```release-note
Stop workspaces prior to updating installations using Single Cluster Ref Arch
```

## Documentation

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2766"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2766"></a>

### CODE REVIEW

**Review by aledbf — APPROVED** [review]

**Review by corneliusludmann — APPROVED** [review]
Looks very good. :pray:

File: gitpod/docs/self-hosted/latest/updating.md [hunk 1]
```diff
@@ -32,16 +34,22 @@ In most cases, updates should work without further effort - and your existing da
...
+
 In this strategy, you have scheduled maintenance windows where you take down your entire Gitpod installation, update it, test it, and then make it available again. Ideally, this is done during times of low demand, e.g. outside of work hours.
 
 ### Secondary Staging Deployment (medium downtime, lower risk, higher cost)
 
+> **Important:** [Single Cluster Reference Architecture](../latest/reference-architecture/single-cluster-ref-arch) is not highly-available, please [stop workspaces](./stop-workspaces) prior to proceeding.
+
 In this strategy, you run the newest version of Gitpod on a secondary (staging) cluster to ensure compatibility/fitness of the newest version within your specific environment. Given that you are testing on a secondary cluster, you save on downtime in your primary cluster. If you are confident in the release, you can then also apply the update to your primary cluster. Given that workspace startups may fail for a brief period _during_ the update process, a maintenance window (albeit smaller) may still be required.
```
Comment (lucasvaltl) [inline gitpod/docs/self-hosted/latest/updating.md:49]:
```suggestion
In this strategy, you run the newest version of Gitpod on a secondary (staging) cluster to ensure compatibility/fitness of the newest version within your specific environment. Given that you are testing on a secondary cluster, you save on downtime in your primary cluster. If you are confident in the release, you can then also apply the update to your primary cluster. Given that workspace startups may fail for a brief period _during_ the update process, a maintenance window (albeit smaller) will still be required.
```

File: gitpod/docs/self-hosted/latest/stop-workspaces.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+title: Stop workspaces
+---
+
+<script context="module">
+  export const prerender = true;
+</script>
+
+# Stop Workspaces
+
+Sometimes it becomes necessary to stop Gitpod workspaces (in a `<namespace>`).
```
Comment (lucasvaltl) [inline gitpod/docs/self-hosted/latest/stop-workspaces.md:13]:
```suggestion
Sometimes it becomes necessary to stop Gitpod workspaces (in a `<namespace>`). Usually this needs to be done when updating Gitpod. Please see the [update guide](./updating) for more information. 
```
Added this just to give more context and allow for an easier navigation.

**Review by lucasvaltl — APPROVED** [review]
Looks great to me - thank you very much for jumping on this! I made some wording suggestions and added another section to the `installing-gitppod.md` guide to also point to the update guide.

**Comment by kylos101** [comment]
Thanks @lucasvaltl , let the build run and take a quick look in preview before merging.

**Comment by kylos101** [comment]
Shipped!

**Comment by kylos101** [comment]
@metcalfc  @hellofromspark FYI

**Pull request merged**